### PR TITLE
Fix #100396: change "note heads" to "noteheads"

### PR DESCRIPTION
**Pull request by ZackTheCardshark**
See http://music.stackexchange.com/questions/42722/notehead-or-note-head

### CODE REVIEW

File: libmscore/line.cpp [hunk 1]
```diff
@@ -542,7 +542,7 @@ QPointF SLine::linePos(Grip grip, System** sys) const
                   if (grip == Grip::START) {
                         cr = static_cast<ChordRest*>(startElement());
                         if (cr && type() == Element::Type::OTTAVA) {
-                              // some sources say to center the text over the note head
+                              // some sources say to center the text over the notehead
                               // others say to start the text just to left of notehead
```
Comment (ZackTheCardshark) [inline libmscore/line.cpp]: These two lines in the comments were amusing.

**Comment by Jojo-Schmitz** [comment]
what about "barline" vs. "bar line"?

**Comment by ZackTheCardshark** [comment]
@Jojo-Schmitz , didn't you already do that one? https://github.com/musescore/MuseScore/pull/2216

**Comment by Jojo-Schmitz** [comment]
Hmm, I think there still or again is a mix. Need to check again

**Comment by Jojo-Schmitz** [comment]
I checked and there is no user visible 'bar line', just in comments, so everything seems to be OK.

**Pull request merged**